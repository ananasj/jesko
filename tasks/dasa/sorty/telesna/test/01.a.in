4
440942239 470555975 594829654 602125929
6
22835069 212882992 417183735 567592923 889061782 904272094
