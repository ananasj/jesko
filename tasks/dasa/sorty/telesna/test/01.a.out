22835069 212882992 417183735 440942239 470555975 567592923 594829654 602125929 889061782 904272094
